src/writer/follow.ts

```diff
@@ -25,6 +25,7 @@ export async function action(message, ipfs, receipt, id): Promise<void> {
     ipfs,
     follower: message.from,
     space: message.space,
+    network: message.network || 's',
```

Comment:
Could we restrict `network` to be values we want?

Reply:
what all networks we want to add this?

Reply:
`network` is now validated